commit 13688137e85c9dc45745214ac9fc618d90836169 (refs/changes/35/2835/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-20T16:22:26-07:00 (2 years ago)
    
    IMGAPI-653 imgapi's sapi_template for ~/.mantaprofile gets MANTA_USER wrong

diff --git a/sapi_manifests/mantaprofile/template b/sapi_manifests/mantaprofile/template
index bd274aa..a807b22 100644
--- a/sapi_manifests/mantaprofile/template
+++ b/sapi_manifests/mantaprofile/template
@@ -1,3 +1,3 @@
-export MANTA_USER={{{ufds_admin_login}}}
+export MANTA_USER={{{IMGAPI_MANTA_USER}}}
 export MANTA_URL={{{IMGAPI_MANTA_URL}}}
 export MANTA_KEY_ID={{{IMGAPI_KEY_ID}}}
